platform/android/export/export.cpp

```diff
@@ -1764,16 +1764,16 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		if (!FileAccess::exists(js)) {
 			valid = false;
-			err += TTR("OpenJDK jarsigner not configured in the Editor Settings.") + "\n";
+			err += TTR("OpenJDK 8 jarsigner not configured in the Editor Settings.") + "\n";
```

Comment:
The documentation says JDK 6 or 8:
https://docs.godotengine.org/en/3.2/getting_started/workflow/export/exporting_for_android.html#setting-it-up-in-godot

Does it actually work only with version 8? Also I don't know if it's already in your plans, but the documentation could also be updated to explicitly say that versions above 8 would fail.

Reply:
I don't know if OpenJDK 6 is still supported. Either way, it's very old and definitely EOL (just like OpenJDK 8, but that one is still commonly used in companies).